Automattic/studio
# Remove unnecessary setTimeout in useExecuteWPCLI hook code block

Related to https://github.com/Automattic/studio/pull/214

## Proposed Changes

- Remove unnecessary timeout that was added to fake the execution time

Sometimes I was receiving an error even the code block was executed correctly. It's hard to replicate and I think it could be related to this overlooked timeout.

## Testing Instructions



- Run Studio with the assistant
- Open the assistant tab
- Send a message to return a wp-cli command, like "generate 10 fake posts"
- Click run
- Observe the code block is still working as expected

## Pre-merge Checklist
